source/beamline/IDEAS/IDEASBeamline.cpp

```diff
@@ -296,6 +296,16 @@ void IDEASBeamline::onShutterValueChanged()
 	emit overallShutterStatus(safetyShutter_->isOpen() && photonShutter2_->isOpen() && safetyShutter2_->isOpen());
 }
 
+AMXRFDetector *IDEASBeamline::xrfDetector() const
+{
+	if (ketek_ )
```

Comment:
Just checking. It looks like this method will preferentially return the Ketek detector. Is there ever a case where there would be instances of both detectors, but the ge13Element is the desired detector?